pkgs/applications/science/astronomy/astrolabe-generator/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchurl, jre, makeWrapper, unzip }:
+
+stdenv.mkDerivation rec {
+  name = "astrolabe-generator-${version}";
+  version = "3.3";
+
+  src = fetchurl {
+    url = "https://github.com/wymarc/astrolabe-generator/releases/download/v3.3/AstrolabeGenerator-3.3.zip";
+    sha256 = "141gfmrqa1mf2qas87qig4phym9fg9gbrcfl2idzd5gi91824dn9";
+  };
+
+  buildInputs = [ jre makeWrapper unzip ];
```

Comment:
`unzip` and `makeWrapper` should be `nativeBuildInputs` as they are only run during the build process.